Xabe.FFmpeg/Streams/AudioStream.cs

```diff
@@ -106,6 +109,12 @@ public IAudioStream SetSeek(TimeSpan? seek)
                 _seek = $"-ss {seek.Value.ToFFmpeg()} ";
             }
             return this;
+        }
+
+        /// <inheritdoc />
+        public AudioCodec GetCodec()
+        {
+            return new AudioCodec(_audio.Replace("-codec:a ", "").Trim(' '));

```

Comment:
Parsing string isn't the best idea ;)
In SetCodec method assign the codec to public property Codec with private set and use this property instead of this method.

Reply:
Done